### PR TITLE
add note to FastBtcDialog.

### PR DESCRIPTION
**Pull request by rick23p**

### CODE REVIEW

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -808,6 +808,7 @@
   "fastBtcDialog": {
     "title": "FastBTC",
     "subtitle": "Adquira rBTC depositando BTC ou realizando uma compra com moeda estatal",
+    "note": "Sempre certifique-se de verificar o endereço de depósito BTC conforme FastBTC é atualizado",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:811]:
```suggestion
    "note": "Verifique sempre se houve mudança no endereço de depósito BTC",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Change on PT-BR

**Review by creed-victor — APPROVED** [review]

**Comment by t-z-u** [comment]
Screenshot 2021-04-27 at 17 49 52

Screenshot 2021-04-27 at 17 49 46


It needs to reflect the design spacing more otherwise we end up with a super tall modal
also lets put the close x on the outside like we are wit the other modals

File: src/locales/pt_br/translation.json [hunk 2]
```diff
@@ -808,6 +808,7 @@
   "fastBtcDialog": {
     "title": "FastBTC",
     "subtitle": "Adquira rBTC depositando BTC ou realizando uma compra com moeda estatal",
+    "note": "Verifique sempre se houve mudança no endereço de depósito BTC",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:811]:
```suggestion
    "note": "Verifique sempre o endereço de depósito BTC",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Change PT-BR

**Comment by peymannaderi3** [comment]
> Screenshot 2021-04-27 at 17 49 52
> 
> Screenshot 2021-04-27 at 17 53 33
> 
> It needs to reflect the design spacing more otherwise we end up with a super tall modal
> also lets put the close x on the outside like we are wit the other modals
> 
> the link has been updated

Reduced the padding
The close button changes are on redesign branch when they are merge it will be fixed.

**Pull request merged**